### DISCUSSION TITLE
When will RCTEventEmitter and RCTModernEventEmitter be replaced and with what?

### DISCUSSION THREAD

**Discussion by BraveEvidence** [reactwg/react-native-new-architecture#66] [Q&A]
Does anyone have any time frame on which `react-native` release will `RCTEventEmitter` and `RCTModernEventEmitter` will be replaced and with what? Both `RCTEventEmitter` and `RCTModernEventEmitter` are deprecated. I have tough time explain my senior dev that there are no alternatives to this.

**Comment by chakrihacker** [comment]
What's the use case?

**Reply by BraveEvidence** [reply]
@chakrihacker Because of this issue https://github.com/reactwg/react-native-new-architecture/discussions/65. Can you please help?
